### PR TITLE
Convert to pyproject.toml

### LINKED ISSUE #6: switch to .toml file for installation

### PR DESCRIPTION
**Pull request by sroet**
closes #6 
I did update some of the other classifiers that are mentioned in https://pypi.org/classifiers/ and added vim swap files to the gitignore

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pytom-template-matching-gpu"
+version = "0.3.4"
+description = "GPU template matching from PyTOM as a lightweight pip package"
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
+    {name = "Marten Chaillet"},
```
Comment (McHaillet) [inline pyproject.toml:12]:
```suggestion
    {name = "Marten Chaillet", email = "m.l.chaillet@uu.nl"},
```
For now we can still add my uu mail as well

**Review by McHaillet — COMMENTED** [review]
Looks good, thanks for the update! 

Should I also give the installation a test?

**Comment by sroet** [comment]
> Should I also give the installation a test?

Please do! also make sure you can install the [plotting] optional as well

**Review by McHaillet — APPROVED** [review]
LGTM, install works like a charm

File: pyproject.toml (hunk 1 above)
Comment (McHaillet) [reply pyproject.toml]: fair enough

**Pull request merged**